### PR TITLE
[HERA-202] End survey fixes

### PR DESCRIPTION
**Pull request by dyudyunov**
- maskot added
- fix for staff view (when there is no answers yet)
- fix for "next" button

**Youtrack:** https://youtrack.raccoongang.com/issue/HERA-202

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

### CODE REVIEW

File: hera_xblocks/end_survey/end_survey/end_survey.py [hunk 1]
```diff
@@ -40,6 +40,10 @@ def confidence(self):
     def title(self):
         return self.data.get("title")
 
+    @property
+    def mascot(self):
```
Comment (sendr) [inline hera_xblocks/end_survey/end_survey/end_survey.py]:
```suggestion
    def mascot_url(self):
```

File: hera_xblocks/end_survey/end_survey/end_survey.py [hunk 2]
```diff
@@ -77,7 +81,9 @@ def result_summary_count(self):
                 }
             }
         }
-        # Count procentage:
+        if not self.result_summary:
```
Comment (sendr) [inline hera_xblocks/end_survey/end_survey/end_survey.py]: you won't get into loop bellow if `self.result_summary` is an empty. I think it's redundant.

File: hera_xblocks/end_survey/end_survey/end_survey.py [hunk 3]
```diff
@@ -139,14 +145,15 @@ def get_context(self):
             "user_result": self.user_result,
             "questions": self.questions,
             "title": self.title,
-            "confidence": self.confidence
+            "confidence": self.confidence,
+            "mascot": self.mascot
```
Comment (sendr) [inline hera_xblocks/end_survey/end_survey/end_survey.py]: mascot_url

File: hera_xblocks/end_survey/end_survey/static/html/end_survey_user.html [hunk 4]
```diff
@@ -1,5 +1,10 @@
 <!-- TODO: Clear the classes because here is alot of junk -->
-<div class="wrapper-page" id="wrapper-page">
-    ${content | n}
+<div class="wrapper-page">
+    <div id="wrapper-page">
+        ${content}
+    </div>
+    %if mascot:
```
Comment (sendr) [inline hera_xblocks/end_survey/end_survey/static/html/end_survey_user.html]: mascot_url

File: hera_xblocks/end_survey/end_survey/static/html/student_completed.html [hunk 5]
```diff
@@ -1,4 +1,4 @@
-<div class="student-complited">
+<div class="student-completed">
```
Comment (sendr) [inline hera_xblocks/end_survey/end_survey/static/html/student_completed.html]: :-)

File: hera_xblocks/end_survey/end_survey/end_survey.py (hunk 2 above)
Comment (dyudyunov) [reply hera_xblocks/end_survey/end_survey/end_survey.py]: There will be a blank page for staff if not to do this check and nobody answered yet.

File: hera_xblocks/end_survey/end_survey/end_survey.py (hunk 1 above)
Comment (dyudyunov) [reply hera_xblocks/end_survey/end_survey/end_survey.py]: done

File: hera_xblocks/end_survey/end_survey/end_survey.py (hunk 3 above)
Comment (dyudyunov) [reply hera_xblocks/end_survey/end_survey/end_survey.py]: done

File: hera_xblocks/end_survey/end_survey/static/html/end_survey_user.html (hunk 4 above)
Comment (dyudyunov) [reply hera_xblocks/end_survey/end_survey/static/html/end_survey_user.html]: done

File: hera_xblocks/end_survey/end_survey/end_survey.py (hunk 2 above)
Comment (dyudyunov) [reply hera_xblocks/end_survey/end_survey/end_survey.py]: fixed

**Pull request merged**